test/runAnalysis/setup.py

```diff
@@ -20,18 +20,22 @@
 #"/store/mc/RunIISummer16MiniAODv2/CITo2E_M1300_CUETP8M1_Lam28TeVDesLR_13TeV_Pythia8_Corrected-v4/MINIAODSIM/PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/70000/20ED2453-E678-E711-9711-0CC47A4C8F30.root",
 #"/store/mc/RunIISummer16MiniAODv2/CITo2E_M1300_CUETP8M1_Lam28TeVDesLR_13TeV_Pythia8_Corrected-v4/MINIAODSIM/PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/70000/84357A56-E678-E711-8F3E-0CC47A4D764C.root",
 #'/store/mc/PhaseIFall16MiniAOD/ZToMuMu_NNPDF30_13TeV-powheg_M_200_400/MINIAODSIM/FlatPU28to62HcalNZSRAW_PhaseIFall16_exo52_90X_upgrade2017_realistic_v6_C1-v1/120000/304E419F-CC13-E711-93E9-FA163E0231A1.root',
-'/store/mc/RunIIFall17MiniAODv2/ZToMuMu_NNPDF31_13TeV-powheg_M_50_120/MINIAODSIM/PU2017_12Apr2018_94X_mc2017_realistic_v14-v2/70000/12BD4CC4-0751-E811-BCA9-0090FAA58D84.root',
+#'/store/mc/RunIIFall17MiniAODv2/ZToMuMu_NNPDF31_13TeV-powheg_M_50_120/MINIAODSIM/PU2017_12Apr2018_94X_mc2017_realistic_v14-v2/70000/12BD4CC4-0751-E811-BCA9-0090FAA58D84.root',
 #'/store/mc/RunIIFall17MiniAOD/MC17_CITo2Mu_M2000toInf_CP5_Lam24TeVDesLRPythia8_v1/MINIAODSIM/PU2017_94X_mc2017_realistic_v11-v1/100000/90F99FA7-E76B-E811-BC35-A0369FD0B362.root'
+#]                          
 
-]                          
 process.maxEvents.input = -1
 isMC = %(isMC)s
 addNTuples = %(addNTuples)s
 year = %(year)d
-process.GlobalTag.globaltag = '%(GT)s'
+#process.GlobalTag.globaltag = '%(GT)s'
```

Comment:
please don'T remove this functionality, we want to specifically choose the GT in the runAnalysis.py!